src/Fields.jl

```diff
@@ -47,3 +47,51 @@ function canonical_unit(x::FieldElement)
   iszero(x) && return one(x)
   return x
 end
+
+@doc raw"""
+    evaluation_points(K::Field, n::Int) -> Vector{FieldElem}
+
+If $K$ contains at least $n$ elements, this function returns a vector $v$ with $n$ distinct elements from $K$. Otherwise it returns an empty vector. 
+"""
+function evaluation_points(K::Field, n::Int)
+   @req n > 0 "n must be positive."
+   if is_finite(K)
+      v = elem_type(K)[]
+      if order(K) < n
+         return v
+      else
+         append!(v, Iterators.take(K, n))
+      end
+   else
+      @assert is_zero(characteristic(K)) "This should not happen. Please open a bug report."
+      v = Vector{elem_type(K)}(undef, n)
+      v[1] = K(div(-n, 2))
+      for i = 2:n
+         v[i] = v[i-1] + 1
+      end
+   end
+   return v
+end
+
+function evaluation_points(K::Generic.RationalFunctionField, n::Int)
```

Comment:
Tests are failing. This method here needs to be moved to the end of `src/generic/RationalFunctionField.jl` and
`evaluation_points` added to the list in `src/generic/imports.jl`.